qa/integration/specs/dlq_spec.rb

```diff
@@ -100,44 +102,41 @@ def generate_message(number)
     end
 
     context 'with multiple pipelines' do
-      let(:message) { generate_message(100)}
       let(:pipelines) {[
           {
               "pipeline.id" => "test",
               "pipeline.workers" => 1,
               "dead_letter_queue.enable" => true,
-              "pipeline.batch.size" => 1,
-              "config.string" => "input { generator { message => '#{message}' codec => \"json\" count => 1000 } } filter { mutate { add_field => { \"geoip\" => \"somewhere\" } } } output { elasticsearch {} }"
+              "pipeline.batch.size" => 100,
```

Comment:
The downsize of the batch size is it functional to the test or to the fact that now it's single node cluster?

Reply:
don't mind my comment, I thought it passed from 100 to 1, but it's opposite and is reasonable to have a batch greater than the unit

Reply:
yeah having size 1 was making these tests unnecessarily slow